packages/webdriverio/src/commands/element/click.ts

```diff
@@ -129,8 +129,14 @@ export default async function click (
                 button
             }]
         }])
-
-        return this.releaseActions()
+        try {
```

Comment:
Can we add a comment as to why we catch this here and ignore the `no such window` error?

Reply:
> Can we add a unit test for this?

Any guidelines for the unit tests? I guess I need to mock the action chains request in `got.ts` but not sure how to write it.

Reply:
Have a look at the [existing unit tests](https://github.com/webdriverio/webdriverio/blob/main/packages/webdriverio/tests/commands/element/click.test.ts). Everything is already mocked and you can just add a test for your use case in a similar fashion. Let me know if anything is unclear.